src/foam/nanos/dig/SUGAR.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.dig',
+  name: 'SUGAR',
+
+  documentation: 'Service Unified GAteway Relay - Perform non-DAO operations against a web service',
```

Comment:
Case error in gateway

Reply:
That's intentional as it is the GA in SUGAR.